docs/learn/learn-staking-advanced.md

```diff
@@ -408,29 +426,34 @@ estimation of all DOT that should be staked, per parachain slot).
 
 :::info The ideal staking rate varies based on the number of parachains
 
-The current staking rate on Polkadot still assumes the absence of parachains, with the suggested 
-ideal staking rate of 75%. You can track the progress on the issue to adjust it 
+The current staking rate on Polkadot still assumes the absence of parachains, with the suggested
+ideal staking rate of 75%. You can track the progress on the issue to adjust it
 [here](https://github.com/paritytech/polkadot/pull/5872). This has already been adjusted on Kusama.
-At the time of updating this section, the ideal staking rate on Kusama was 50.5% when there were 
-49 parachain slots. When the number of slots goes to 60, the ideal staking rate will be 45%. 
-[Here](https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L535)
-is the code for reference. This code assumes that the number of slots auctioned correspond to the number of parachains on the relaychain, which may not be true as new slots can be occupied by old parachains that are renewing their lease. You can also track the progress on resolving this specific issue 
+At the time of updating this section, the ideal staking rate on Kusama was 50.5% when there were 49
```

Comment:
Maybe use a date here instead for clarity.

Reply:
Now that I think about it, probably better to remove this dependence on the date of writing the section. I rewrote  it in more general terms.